sirepo/sim_data/ml.py

```diff
@@ -104,30 +103,27 @@ def __init__(self, file_path):
     def _get_archive_extension(self):
         x = list(
             filter(
-                lambda s: self._is_archive_type(s),
+                lambda s: self.path.basename.endswith(s),
```

Comment:
You can say `self.path.ext == f".{s}"` which I think is more explicit about what you're after.

Reply:
That won't work for ".tar.gz"; its `.ext` is ".gz", which by itself is not an archive (just compression). We must explicitly test for .tar.gz.